### PR TITLE
Add link to Ansible Integration document

### PR DESCRIPTION
**Pull request by lauranovich**
Adds a link to the Ansible Integration document project on GitHub

### CODE REVIEW

File: docs/source/install/index.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@ Download and Install Scylla Monitoring Stack
    Docker Compose <docker_compose>
    System Recommendations <min-prod-hw>
    Using Thanos <thanos>
-
+   Integrate Scylla with Ansible <https://github.com/scylladb/scylla-ansible-roles/wiki/Deploying-Scylla-Monitoring-and-connecting-it-to-a-Scylla-Cluster>
```
Comment (tzach) [inline docs/source/install/index.rst]:
```suggestion
   Install Scylla with Ansible <https://github.com/scylladb/scylla-ansible-roles/wiki/Deploying-Scylla-Monitoring-and-connecting-it-to-a-Scylla-Cluster>
```

File: docs/source/install/index.rst [hunk 2]
```diff
@@ -23,3 +23,4 @@ Choose a topic to begin:
 * :doc:`Deploy Scylla Monitoring Stack Without Docker <monitor_without_docker>`
 * :doc:`Running using Docker Compose <docker_compose>`
 * :doc:`Using Thanos <thanos>`
+* `Integrate Scylla with Ansible <https://github.com/scylladb/scylla-ansible-roles/wiki/Deploying-Scylla-Monitoring-and-connecting-it-to-a-Scylla-Cluster>`_
```
Comment (tzach) [inline docs/source/install/index.rst]:
```suggestion
* `Install Scylla with Ansible <https://github.com/scylladb/scylla-ansible-roles/wiki/Deploying-Scylla-Monitoring-and-connecting-it-to-a-Scylla-Cluster>`_
```

**Comment by tzach** [comment]
I missed the fact this is a scylla monitoring PR!
Why do we need Ansible here?

**Comment by lauranovich** [comment]
I am providing the link because I thought it was good to have. If this is not the case close the PR
@tzach your call.

**Pull request closed without merging**